syntax = "proto3";

package streaming;

message LogRequest {
  string process_name = 1;
}

message LogEntry {
  string message = 1;
  int32 line_number = 2;
}

service LogService {
  rpc StreamLogs(LogRequest) returns (stream LogEntry);
}
